src/Nethermind/Nethermind.Serialization.Json/EthereumJsonSerializer.cs

```diff
@@ -128,14 +128,14 @@ public void RegisterConverter(JsonConverter converter)
             RebuildSerializers();
         }
 
-        private void RebuildSerializers()
+        private void RebuildSerializers(int? maxDepth = null)
         {
             _readableSettings = new JsonSerializerSettings
             {
                 ContractResolver = new CamelCasePropertyNamesContractResolver(),
                 NullValueHandling = NullValueHandling.Ignore,
                 Formatting = Formatting.Indented,
-                Converters = ReadableConverters
+                Converters = ReadableConverters,
```

Comment:
super minor cosmetics - redundant comma. Can be safely ignored

Reply:
I like those commas, if you add new line you don't have to change this one

Reply:
Yes, all language should allow this.